canton/it-lib/src/main/com/daml/CantonRunner.scala

```diff
@@ -172,6 +172,7 @@ object CantonRunner {
               config.adminToken,
               ApiTypes.ApplicationId("CantonRunner"),
             )
+            // TODO: better to sequence the dar-loading in case of dependencies
```

Comment:
```suggestion
            // TODO https://github.com/digital-asset/daml/issues/16151
            // better to sequence the dar-loading in case of dependencies
```

just so we don't miss it

Reply:
ok